### ISSUE TITLE
Feature Request: Play sound as alarm sound

### ISSUE THREAD

**Issue by meisterm** [phonegap/phonegap-plugin-push#1176]
Hi all,

could it be possible to play a sound like an alarmsound. So the sound will be played with the volume of the alarm volume like the alarm clock?

Regards,
meisterm

**Comment by macdonst** [comment]
No, it is not possible. The volume of the sound played when a push notification is received is controlled by the OS as set by the user.

**Comment by meisterm** [comment]
Yes. But there is definitly the possibility to use a different stream for sound playback. https://developer.android.com/reference/android/media/AudioManager.html
STREAM_ALARM is used for Alarmsounds which will be played with a different volume. (It's possible to define different volumes) So you can have a muted Soundvolume or Ringtonevolume and a loud alarmvolume for your alarm clock for example.
The plugin should play with the STREAM_ALARM if there is set a special flag in the payload. That's my wish :-)

**Comment by macdonst** [comment]
@meisterm I spoke too quickly there is a way to set the sound with AudioAttributes that can be used to indicate an alarm.

**Comment by markterrill** [comment]
I am very very interested in this +1001 ;)
